### PR TITLE
support github.com/nats-io/nats-server/v2 and github.com/nats-io/nats.go

### PR DESCRIPTION
**Pull request by tsingson**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
 
-# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
-.glide/
+# build
+/cmd/comet/comet
+/cmd/logic/logic
+/cmd/job/job
+/target
+/configs
+/dist
```
Comment (tylertreat) [inline .gitignore]: What is this stuff for?

File: go.mod [hunk 2]
```diff
@@ -5,30 +5,29 @@ go 1.12
...
-	github.com/liftbridge-io/nats-on-a-log v0.0.0-20180718011723-80d0727461af
 	github.com/natefinch/atomic v0.0.0-20150920032501-a62ce929ffcc
-	github.com/nats-io/gnatsd v1.3.0
-	github.com/nats-io/go-nats v1.7.0
-	github.com/nats-io/nkeys v0.0.2 // indirect
-	github.com/nats-io/nuid v1.0.0
+	github.com/nats-io/gnatsd v1.3.0 // indirect
+	github.com/nats-io/go-nats v1.7.0 // indirect
+	github.com/nats-io/nats-server/v2 v2.0.0
+	github.com/nats-io/nats.go v1.8.1
```
Comment (tylertreat) [inline go.mod]: Why are both gnatsd/nats-server and go-nats/nats.go listed?

File: server/raft.go [hunk 3]
```diff
@@ -11,8 +11,9 @@ import (
 
 	"github.com/hashicorp/raft"
 	"github.com/hashicorp/raft-boltdb"
-	"github.com/liftbridge-io/nats-on-a-log"
-	"github.com/nats-io/go-nats"
+	"github.com/nats-io/nats.go"
+
+	"github.com/liftbridge-io/liftbridge/third/nats-on-a-log"
```
Comment (tylertreat) [inline server/raft.go]: Why move this under `/third`?

Comment (tsingson) [reply server/raft.go]: > Why move this under `/third`?

sorry about this. 
it's for my personal test ,  forget to fix.

**Pull request closed without merging**